src/Drawable.js

```diff
@@ -720,14 +720,6 @@ class Drawable {
      */
     static sampleColor4b (vec, drawable, dst, effectMask) {
         const localPosition = getLocalPosition(drawable, vec);
-        if (localPosition[0] < 0 || localPosition[1] < 0 ||
-            localPosition[0] > 1 || localPosition[1] > 1) {
-            dst[0] = 0;
-            dst[1] = 0;
-            dst[2] = 0;
-            dst[3] = 0;
-            return dst;
-        }
```

Comment:
So... it turns out I steered you wrong when I recommended removing this. I was thinking about texture coordinates being out of range (which should be and is handled at the Silhouette layer) when I should have been thinking about geometric coordinates being out of range (which is what this `if` handles). I think bringing this back in will fix the crab test.